config/tomlConfig_test.go

```diff
@@ -678,6 +678,12 @@ func TestEnableEpochConfig(t *testing.T) {
     # AlwaysSaveTokenMetaDataEnableEpoch represents the epoch when the token metadata is always saved
     AlwaysSaveTokenMetaDataEnableEpoch = 61
 
+		# RuntimeCodeSizeFixEnableEpoch represents the epoch when the code size fix in the VM is enabled
```

Comment:
indentation

Reply:
space instead of tabs

Reply:
Replaced with spaces inside test strings.